imagery/i.atcorr/altitude.cpp

```diff
@@ -382,6 +382,8 @@ void Altitude::parse()
     cin >> original_xpp;
     cin.ignore(numeric_limits<int>::max(), '\n'); /* ignore comments */
     original_xpp = -original_xpp;
+    printf("Original Alt: XPS = %.2f\t XPP = %.2f\n", original_xps,
```

Comment:
Please consider using `G_message()` (see https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting_c.md#messages).

Reply:
There is a G_Debug() maybe?

Reply:
Yes, using `G_debug()` seems perhaps even more appropriate in this case
(https://grass.osgeo.org/programming8/debug_8c.html#af8982bd66c95e3d8d2773ed956c20733).

Reply:
Hi what do I do now?

Reply:
Use `G_debug()` :-)